pages/index.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+    <section-hero />
+    <section-project />
+    <section-slider />
```

Comment:
You don't need to create components for each part.
You only need to create components if they are repeated in multiple places.
If all these sections are not repeated on some other page, let's put them directly here.